src/Mixer/Mixer.cpp

```diff
@@ -0,0 +1,109 @@
+#include "NAS2D/Mixer/Mixer.h"
+
+namespace NAS2D
+{
+
+	Mixer::Mixer(const std::string& name)
+	: mName(name)
+	{
+	}
+
+	Mixer::Mixer()
+	{
+	}

```

Comment:
For empty constructors, it may be better to use `= default` rather than `{}`. The empty `{}` makes the class non-trivial, 

References:
https://en.cppreference.com/w/cpp/types/is_trivial
https://en.cppreference.com/w/cpp/language/copy_constructor#Trivial_copy_constructor

Reply:
Whups, that was an oversight.